src/harness/sas_test_harness.py

```diff
@@ -369,7 +369,7 @@ def generateCbsdRecords(registration_requests, grant_requests_list):
                                    registration_request['cbsdSerialNumber'])
 
     internal_data = {
-        'id': cbsd_reference_id,
+        'id': 'cbsd/' + cbsd_reference_id,
```

Comment:
These changes make sense (thank you). Is this the only place where `generateCbsdReferenceId` is called?